examples/resources/awscc_efs_access_point/awscc_efs_access_point_posix_user.tf

```diff
@@ -0,0 +1,32 @@
+resource "awscc_efs_access_point" "this" {
+  file_system_id = awscc_efs_file_system.this.id
+  posix_user = {
+    gid = 0
+    uid = 0
```

Comment:
Ideally let's not use root for this

can you change the gid and uid to something higher than 1000? 

i.e. 1001